package.json

```diff
@@ -12,6 +12,7 @@
     "format:package_json": "lerna exec -- prettier-package-json",
     "postinstall": "husky",
     "lint": "npm run lint:css && npm run lint:js && npm run lint:md",
+    "lint:ci": "npm run lint && npm run format:all && npm exec tsc",
```

Comment:
Spliting linting from all tests to speed up workflow.

Reply:
After the `.circleci/config.yml` is restored, please be sure to include `lint:ci` in the test portion of that workflow.